Formula/portable-openssl.rb

```diff
@@ -39,6 +39,7 @@ def arch_args
       {
         :x86_64 => %w[linux-x86_64],
         :i386  => %w[linux-generic32],
+        :arm => %w[linux-armv4],
```

Comment:
I do have a Raspberry Pi 1, but limited free time at home with two kids about. Once you think it's ready for a test on real hardware, I can take a look at it.